src/components/troubleshooting-card/index.tsx

```diff
@@ -9,23 +9,17 @@ import type { TroubleshootingDataElement } from 'utils/typings/types'
 export default function TroubleshootingCard({
   title,
   slug,
-  tags,
+  productTeam,
```

Comment:
This is really suposed to be `tags`. In the current help-center-content state we must still run the script that puts them into the frontmatter. Then they'll be rendered.